src/shared/src/components/organisms/AltinnTable.tsx

```diff
@@ -1,29 +1,41 @@
-import { Grid, makeStyles, Table, TableContainer } from '@material-ui/core';
+import {
+  Grid,
+  makeStyles,
+  Table,
+  TableContainer,
+  TableProps,
+} from '@material-ui/core';
 import React from 'react';
 
 export interface IAltinnTableProps {
-  children: React.ReactNode;
+  tableLayout?: 'fixed' | 'auto';
+  wordBreak?: 'break-word' | 'normal';
   id: string;
 }
 
-const useStyles = makeStyles({
-  table: {
-    tableLayout: 'fixed',
-    marginBottom: '12px',
-    wordBreak: 'break-word',
-  },
+const useStyles = makeStyles(() => {
+  return {
+    table: ({
+      tableLayout = 'fixed',
+      wordBreak = 'break-word',
+    }: IAltinnTableProps) => {
+      return {
+        tableLayout: tableLayout,
+        marginBottom: '12px',
+        wordBreak,
+      };
+    },
+  };
 });
 
-export default function AltinnTable(props: IAltinnTableProps) {
-  const {
-    id, children,
-  } = props;
-  const classes = useStyles();
+export default function AltinnTable(
+  props: IAltinnTableProps & Omit<TableProps, 'id'>,
+) {
+  const { tableLayout, wordBreak, ...tableProps } = props;
```

Comment:
See warnings below

Reply:
Ah, ok, I was a bit quick here and didn't realize that's what that line was doing 😅 . Yep, then you can go ahead and update eslint config with the suggestion above :)